### PR TITLE
update to clear shit up

### PR DESCRIPTION
**Pull request by Zoardedz**

### CODE REVIEW

**Review by MemeHoovy — APPROVED** [review]

**Review by TheKitBoi — APPROVED** [review]

**Comment by ACrazyTown** [comment]
Is `character.updateAnimation()` necessary? Iirc it worked fine for me without it

**Comment by Cheemsandfriends** [comment]
`character.updateAnimation()` doesn't need to be called, it's something that FlxSprite already uses.
The only thing you could say I'm doing is just replacing the animation handlers.
Why does it not work for you? Are you using a class extending `FlxAnimate` and overriding update without adding `super.update()`?
that would be my guess

**Comment by Zoardedz** [comment]
im not extending flxanimate, the updateAnimation function was necessary for me because it didn't play the animation

File: README.md [hunk 1]
```diff
@@ -69,6 +69,16 @@ Adding animations from indices:
 character.anim.addByAnimIndices(AnimationName, Indices ([0, 1, 2, 3...] etc.), Framerate);
 ```
 
+After doing all of that play the animation as follows:
+```haxe
+character.anim.play(AnimationName);
+```
+
+To actually let the animation update go to the update function and do:
```
Comment (TheKitBoi) [inline README.md:77]: you should add the (optional)

**Review by TheKitBoi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Zoardedz) [reply README.md:77]: mk

**Comment by Zoardedz** [comment]
i made a file which basically prevents it from doing this, it just gets a spriteatlas group then pushes itself to that group, you can for loop in the update function on each spriteatlas in the current state you're in and by that i dont mean the spriteatlas itself and do the update function for it 
[SpriteAtlas.txt](https://github.com/Dot-Stuff/flxanimate/files/9941050/SpriteAtlas.txt)

**Comment by ACrazyTown** [comment]
> i made a file which basically prevents it from doing this, it just gets a spriteatlas group then pushes itself to that group, you can for loop in the update function on each spriteatlas in the current state you're in and by that i dont mean the spriteatlas itself and do the update function for it [SpriteAtlas.txt](https://github.com/Dot-Stuff/flxanimate/files/9941050/SpriteAtlas.txt)

But the thing is that this shouldn't happen at all. `updateAnimation()` should not be needed to be called, I have never needed to call it and I've been using FlxAnimate since like early 2022.

How do you intialise FlxAnimate? Are you sure you don't set `active` to false or something?

**Comment by Zoardedz** [comment]
yes i'm sure of all of that.

**Comment by Cheemsandfriends** [comment]
I really can not merge this pull request, since this isn't what a dev who wants to implement FlxAnimate that doesn't want to touch much about the source code has to add nowhere.
`updateAnimation()` is an FlxSprite function that is already been called in [`update()`](https://github.com/HaxeFlixel/flixel/blob/05b41665697e54f6571e0452ad6b5480ebf01df5/flixel/FlxSprite.hx#L751). The only thing that FlxAnimate does to redirect the animation handler is overriding the function and call the update function of the animation handler in there.
But I do wanna know why you're having this issue, that's been bothering me for a while.

**Comment by Zoardedz** [comment]
i honestly don't know but close this pull request if you want, if somebody else is having this problem i'll see if i can help

**Pull request closed without merging**